README.md

```diff
@@ -279,18 +279,18 @@ Open `x64 Native Tools Command Prompt for VS2019` and run the following commands
 ``` batch
 > git config --global core.eol lf
 > git config --global core.autocrlf input
-> set OPENMODELICAHOME=
-> set OPENMODELICALIBRARY=
 > md dynawo-project && cd dynawo-project
-> git clone https://github.com/dynawo/dynawo.git dynawo
+> git clone https://github.com/dynawo/dynawo.git
 > cd dynawo
-> cmake -S dynawo/3rdParty -B b-3-p -DCMAKE_INSTALL_PREFIX=../d-3-p -DOPENMODELICA_INSTALL=../OpenModelica/Install -DOPENMODELICA_SRC=../OpenModelica/Source -DOMDEV_HOME=../OMDev -G "NMake Makefiles"
-> cmake --build b-3-p
-> cmake -S dynawo -B b -DCMAKE_INSTALL_PREFIX=../d-i -DDYNAWO_HOME=.. -DINSTALL_OPENMODELICA=../../OpenModelica/Install -DDYNAWO_THIRD_PARTY_DIR=../../d-3-p -G "NMake Makefiles
-> cmake --build b --target install
-> cmake --build b --target models
-> cmake --build b --target solvers
+> util\windows\dynawo.cmd build
 ```
+**Tip** If you have already OpenModelica installed in another Dynawo workspace, it is possible to set the following environment variables to use the existing OpenModelica installation and avoid recompiling OpenModelica again :
+``` batch
+> set OPENMODELICA_INSTALL=PATH_TO_OPENMODELICA_INSTALL
```

Comment:
We also need a section in which we describe how to launch a simulation

Reply:
We need to complete this https://github.com/dynawo/dynawo?tab=readme-ov-file#windows-3 ?

Reply:
so I took a second look and I propose that
- we double check the jobs commands given in the README work as expected
- we also need to make sur ethe documentation is allright (I'm quite sure it's not)